src/main/java/org/triplea/ui/TripleA.java

```diff
@@ -0,0 +1,104 @@
+package org.triplea.ui;
```

Comment:
@RoiEXLab Sorry if you already responded to this comment--I couldn't find it.

I think @DanVanAtta's motivation behind this comment is spot on.  At the highest level of package structure, we should be thinking about modules that could potentially be built and distributed separately.  For example, there should be no need to distribute the JFX client with a headless server.  I realize actually achieving this is a long way off, but starting with the right package names will go a long way to help us think about the "shape" of the various pieces of TripleA.  So I think `org.triplea.client.*` is a good start.

Not really for this PR, but let me ask you something related...  The Swing and JFX clients will obviously exist side-by-side for some time.  Do you expect the core Swing UI to be part of the forthcoming mass package renaming?  If so, do we need to think about distinguishing between multiple clients in this repo?  For example, `org.triplea.client.javafx.*` and `org.triplea.client.swing.*`?